.chloggen/rm_opencensus_receiver_parsing.yaml

```diff
@@ -0,0 +1,18 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: operator
```

Comment:
I think (not sure) we are marking issues like this with `collector`.

Reply:
```suggestion
component: collector
```